### ISSUE TITLE
Toggle Led

### ISSUE THREAD

**Issue by jgelon** [mac-zhou/midea-ac-py#74]
It would be very nice to be able to either toggle the led display to on/off or to use something similar as the tone prompt in the config.

Reason we have the ac on the bedroom and at night the led is too bright.

**Comment by rokam** [comment]
I don't think that is possible, I did search that in the doc and I wasn't able to find a flag for this. Although I do think that would be great.

**Comment by mac-zhou** [comment]
I will add 1 configuration item

**Comment by yevon** [comment]
Very nice, I was about to ask the same.

**Comment by henrywinn** [comment]
Was this ever implemented? Is there anything I can do to get this moving?

**Comment by tinuva** [comment]
Also would love this function.

I have also not seen this in the midea air app either but still hoping!

**Comment by amoyzc** [comment]
> I will add 1 configuration item

Are we going to get this? Another reason why this is needed is when the led turn off, it will also mute the AC as well. Which is very useful for automations.

**Comment by mill1000** [comment]
There's a PR out for this #177. 

I have a fork with this and other PRs merged. You can add it as a custom repo in HACS.
https://github.com/mill1000/midea-ac-py
